### DISCUSSION TITLE
Moon Raid

### DISCUSSION THREAD

**Discussion by sheazywi** [TeamGalacticraft/Galacticraft#299] [Feature Requests]
As you can see here below in the image 
![image](https://github.com/TeamGalacticraft/Galacticraft/assets/73042839/5c4265bc-4648-4403-b0d2-9efb7f6f4b32)
You can see that Pillager Tower and Moon Village are imported, I had an idea while looking at the new commit made on the 24th January. Why not make Moon raids using the same mechanics as the one on the overworld. Maybe change it a little bit but why not.

**Comment by vos6434** [comment]
Raids are planned but not currently being worked on.
